ChameleonForms.Example/Views/ExampleForms/_ChildPartial.cshtml

```diff
@@ -0,0 +1,12 @@
+﻿@model ChildViewModel

```

Comment:
This shows what the partial itself looks like.

Note: you can determine if you are within a section or not and react appropriately.